### ISSUE TITLE
`mkdir /tmp/blobfuse` on node required for working installation

### ISSUE THREAD

**Issue by mikemonteith** [Azure/kubernetes-volume-drivers#58]
**What happened**:

I followed the README instructions to install blobfuse on kubernetes (AKS).

My pod would not start due to errors
```
$ kubectl describe pod nginx-flex-blobfuse

...
Events:
  Type     Reason       Age                  From                                         Message
  ----     ------       ----                 ----                                         -------
  Warning  FailedMount  42m (x13 over 132m)  kubelet, aks-devuksouth-26257669-vmss000002  Unable to attach or mount volumes: unmounted volumes=[test], unattached volumes=[default-token-c5xpw test]: timed out waiting for the condition
  Warning  FailedMount  17m (x38 over 135m)  kubelet, aks-devuksouth-26257669-vmss000002  Unable to attach or mount volumes: unmounted volumes=[test], unattached volumes=[test default-token-c5xpw]: timed out waiting for the condition
  Warning  FailedMount  10m (x70 over 137m)  kubelet, aks-devuksouth-26257669-vmss000002  MountVolume.SetUp failed for volume "test" : invalid character 'E' looking for beginning of value
```

I followed the troubleshooting guide by inspecting the node logs with `tail -f /var/log/blobfuse-driver.log`

```
Wed Jun 10 14:06:29 UTC 2020 EXEC: mkdir -p /var/lib/kubelet/pods/143d746f-a5e6-4723-9b35-fe9a692df06d/volumes/azure~blobfuse/test
Wed Jun 10 14:06:29 UTC 2020 INF: AZURE_STORAGE_ACCESS_KEY is set 
Wed Jun 10 14:06:29 UTC 2020 INF: export storage account - export AZURE_STORAGE_ACCOUNT=mmonteithblobtest 
Wed Jun 10 14:06:29 UTC 2020 EXEC: blobfuse /var/lib/kubelet/pods/143d746f-a5e6-4723-9b35-fe9a692df06d/volumes/azure~blobfuse/test --container-name=test --tmp-path=/tmp/blobfuse -o allow_other  --file-cache-timeout-in-seconds=120
Wed Jun 10 14:06:29 UTC 2020 ERROR: { "status": "Failure", "message": "Failed to mount device /dev/ at /var/lib/kubelet/pods/143d746f-a5e6-4723-9b35-fe9a692df06d/volumes/azure~blobfuse/test, accountname:mmonteithblobtest, error log:Wed Jun 10 14:06:29 UTC 2020 EXEC: blobfuse /var/lib/kubelet/pods/143d746f-a5e6-4723-9b35-fe9a692df06d/volumes/azure~blobfuse/test --container-name=test --tmp-path=/tmp/blobfuse -o allow_other --file-cache-timeout-in-seconds=120" }
```

After some investigation, I found that the /tmp/blobfuse directory didn't exist and it needed to be created.

After running `mkdir /tmp/blobfuse` on the node, my pod started working

```
Wed Jun 10 14:08:31 UTC 2020 EXEC: mkdir -p /var/lib/kubelet/pods/143d746f-a5e6-4723-9b35-fe9a692df06d/volumes/azure~blobfuse/test
Wed Jun 10 14:08:31 UTC 2020 INF: AZURE_STORAGE_ACCESS_KEY is set 
Wed Jun 10 14:08:31 UTC 2020 INF: export storage account - export AZURE_STORAGE_ACCOUNT=mmonteithblobtest 
Wed Jun 10 14:08:31 UTC 2020 EXEC: blobfuse /var/lib/kubelet/pods/143d746f-a5e6-4723-9b35-fe9a692df06d/volumes/azure~blobfuse/test --container-name=test --tmp-path=/tmp/blobfuse -o allow_other  --file-cache-timeout-in-seconds=120
Wed Jun 10 14:08:31 UTC 2020 INFO: {"status": "Success"}
```

**What you expected to happen**:

/tmp/blobfuse should not need to be manually created

**Environment**:
- Kubernetes AKS cluster (v1.16.9)
- Linux

**Comment by andyzhangx** [comment]
> Looks like this bug has already been fixed in blobfuse master, but not released yet.
> https://github.com/Azure/azure-storage-fuse/pull/413/files

correct, blobfuse `1.2.3` is breaking, to mitigate this issue, I think we could `mkdir /tmp/blobfuse` in our flexvolume blobfuse driver side

**Comment by andyzhangx** [comment]
this issue is fixed by `blobfuse-flexvolume:1.0.13`, try with 
```
kubectl apply -f https://raw.githubusercontent.com/Azure/kubernetes-volume-drivers/master/flexvolume/blobfuse/deployment/blobfuse-flexvol-installer-1.9.yaml
```
